plugins/router/snmp__mikrotik_flash

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/perl -w
+#
```

Comment:
Please use a perldoc header here - this allows automatic man page generation and better visualization in the plugin gallery.

Reply:
I'm not sure how to use perldoc. It's like the headers in other plugins?

Reply:
Yes. Just look for `=cut` in other plugins.

Reply:
I've added the header, I hope it's ok.